aten/src/ATen/native/native_functions.yaml

```diff
@@ -940,7 +940,7 @@
 - func: as_strided(Tensor(a) self, SymInt[] size, SymInt[] stride, SymInt? storage_offset=None) -> Tensor(a)
   variants: function, method
   dispatch:
-    ZeroTensor, CPU, CUDA: as_strided_tensorimpl
+    ZeroTensor, CPU, CUDA, MkldnnCPU: as_strided_tensorimpl
```

Comment:
mkldnn tensor is opaque without the definition of "stride". It is weird to provided an implementation as `as_strided` for it. It must be something wrong from the caller.

Reply:
Removed